src/assemble_workflow/dist.py

```diff
@@ -13,11 +13,13 @@
 from abc import ABC, abstractmethod
 
 from assemble_workflow.bundle_rpm import BundleRpm
+from manifests.build_manifest import BuildManifest
 from system.zip_file import ZipFile
 
 
 class Dist(ABC):
-    def __init__(self, name: str, path: str, min_path: str) -> None:
+    def __init__(self, name: str, path: str, min_path: str, build_kls: BuildManifest.Build) -> None:
+        self.build_kls = build_kls
```

Comment:
What is "kls"? You mean class? That's `cls` in Python.

Reply:
Valid call outs.

Reply:
Removed kls, now it is using the dict as `build_dict`.